src/Mayaqua/CMakeLists.txt

```diff
@@ -19,11 +19,24 @@ find_package(ZLIB REQUIRED)
 # In some cases libiconv is not included in libc
 find_library(LIB_ICONV iconv)
 
-add_subdirectory(cpu_features)
+if (HAVE_SYS_AUXV)
+  add_subdirectory(cpu_features)
+endif ()
 
 target_include_directories(mayaqua PRIVATE cpu_features/include)
 
-target_link_libraries(mayaqua cpu_features OpenSSL::SSL OpenSSL::Crypto Threads::Threads ZLIB::ZLIB)
+target_link_libraries(mayaqua OpenSSL::SSL OpenSSL::Crypto Threads::Threads ZLIB::ZLIB)
+
+if (HAVE_SYS_AUXV)
+  target_link_libraries(mayaqua cpu_features)
+else (HAVE_SYS_AUXV)
+  add_definitions(-DSKIP_CPU_FEATURES)
+endif (HAVE_SYS_AUXV)
+
+find_library(LIBRT_LIBRARIES rt)
+if(LIBRT_LIBRARIES)
+  target_link_libraries(mayaqua rt)
+endif(LIBRT_LIBRARIES)
```

Comment:
I think that there's no need to specify the condition after `else` and `endif`, as the block is very small.